### PR TITLE
DCGAN_tensorflow

### PR DESCRIPTION
**Pull request by shvmshri**

### CODE REVIEW

File: generative_models/DCGAN_Tensorflow/main.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    fake_loss = cross_entropy(tf.zeros_like(fake_output), fake_output)
+    total_loss = real_loss + fake_loss
+    return total_loss
+
+def generator_loss(fake_output):
+    return cross_entropy(tf.ones_like(fake_output), fake_output)
+
+# Defining optimizers
+generator_optimizer = Adam(learning_rate=0.0002)  
+discriminator_optimizer = Adam(learning_rate=0.0002)
```
Comment (nirmal-suthar) [inline generative_models/DCGAN_Tensorflow/main.py]: add learning rate in parser

File: generative_models/DCGAN_Tensorflow/main.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+# Saving Checkpoints
+checkpoint_dir = './training_checkpoints'
```
Comment (nirmal-suthar) [inline generative_models/DCGAN_Tensorflow/main.py]: checkpoint dir as parser

File: generative_models/DCGAN_Tensorflow/main.py (hunk 1 above)
Comment (nirmal-suthar) [reply generative_models/DCGAN_Tensorflow/main.py]: add as two different lr parameter for gen and dis

**Comment by nirmal-suthar** [comment]
update readme (help log) after adding new parser arguments

File: generative_models/DCGAN_Tensorflow/main.py (hunk 1 above)
Comment (shvmshri) [reply generative_models/DCGAN_Tensorflow/main.py]: done

File: generative_models/DCGAN_Tensorflow/main.py (hunk 2 above)
Comment (shvmshri) [reply generative_models/DCGAN_Tensorflow/main.py]: done
